src/Sylius/Bundle/CoreBundle/Application/Kernel.php

```diff
@@ -97,17 +97,14 @@ public function registerBundles(): array
             new \Sylius\Bundle\FixturesBundle\SyliusFixturesBundle(),
             new \Sylius\Bundle\PayumBundle\SyliusPayumBundle(), // must be added after PayumBundle.
             new \Sylius\Bundle\ThemeBundle\SyliusThemeBundle(), // must be added after FrameworkBundle
+
+            new \Symfony\Bundle\WebServerBundle\WebServerBundle(), // allows to run build in web server. Not recommended for prod environment
```

Comment:
why moving this here ?

Reply:
We decided that it does not bring much optimization but is useful when you have an issue appearing only on prod env. We still run apps in prod locally from time to time to discover a problem.

Reply:
To make it available in another environments like staging (and nobody's forced to use it on prod ofc, it's just an easy way to see if things are working correctly without setting up a webserver).

Reply:
ok I understand the reason. I was just surprised cause I've seen the previous pull-request moving it on dev/test environment :)